src/platform-includes/session-replay/setup/javascript.mdx

```diff
@@ -1,7 +1,7 @@
-To set up the integration, add the following to your Sentry initialization. Several options are supported and passable via the integration constructor. See the [configuration sections](/platforms/javascript/session-replay/) for more details.
+To set up the integration, add the following to your Sentry initialization. Several options are supported and passable using the integration constructor. See the [configuration sections](/platforms/javascript/session-replay/configuration/) for more details.
```

Comment:
This link was pointing to the same page. I'm not sure if that's intentional (seems odd) or if this was meant to point to the configuration content.